bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx64-base.html

```diff
@@ -0,0 +1,148 @@
+  {% set hero_desc = _('One log-in gives you access to all these benefits') %}
+  {% set button_create = _('Create a Firefox Account') %}
+  {% set login_desc = _('Have an account?') %}
+  {% set login_link = _('Log in') %}
+  {% set notes_head = _('Pick up where you left off with Notes') %}
+  {% set notes_desc = _('Your ideas and inspiration are secure and encrypted with Notes – and when you’re signed into your Account, they sync from your desktop to your Android devices.') %}
+  {% set notes_link = _('Get the Notes App') %}
+  {% set lockbox_head = _('Remember your passwords in Lockbox') %}
+  {% set lockbox_desc = _('No more getting locked out of apps and websites. Lockbox secures all the passwords you’ve saved in Firefox and gives you easy access across all your iOS devices.') %}
+  {% set lockbox_link = _('Get the Lockbox App') %}
+  {% set pocket_head = _('Put quality content in your Pocket') %}
+  {% set pocket_desc = _('Discover the web’s best content, and absorb it anytime – even offline – on any device. Pocket’s listen feature will even read any article aloud to you. Get it all from the Firefox toolbar or the Pocket app.') %}
+  {% set pocket_link = _('Get the Pocket App') %}
+  {% set sync_head = _('Send tabs instantly to your devices') %}
+  {% set sync_desc = _('Send an open tab on one device to all your others with a single tap. Much easier than texting or emailing yourself those links.') %}
+{% endif %}
+
+{% block page_title %}{{ page_title }}{% endblock %}
+{% block page_desc %}{{ page_desc }}{% endblock %}
+
+{% block page_css %}
+ {{ css_bundle('firefox_whatsnew_64') }}
+{% endblock %}
+
+{% block site_header %}{% endblock %}
+
+{% block content %}
+<main class="mzp-t-firefox">
+  <header class="c-page-header">
+    <div class="mzp-l-content c-page-header-inner">
+      {{ high_res_img('logos/firefox/logo-quantum-wordmark-large.png', {'alt': 'Firefox', 'width': '147', 'height': '55', 'class': 'c-page-header-logo-fx'}) }}
+      <h1 class="c-page-header-up-to-date">{{ latest }}</h1>
+      <img src="{{ static('img/logos/mozilla/wordmark-dark.svg') }}" alt="Mozilla" width="112" height="32" class="c-page-header-logo-moz">
+    </div>
+  </header>
+  {% set content_blocking_image='firefox/whatsnew_64/hero.png' if DIR == 'ltr' else 'firefox/whatsnew_64/hero-rtl.png' %}
+  {% call hero(
+   title=hero_title_out,
+   desc=hero_desc,
+   class='mzp-has-image mzp-l-reverse t-wn64',
+   image_url=content_blocking_image,
+   include_highres_image=true,
+   include_cta=True
+  ) %}
+  <p>
+    <a href="https://accounts.firefox.com/signup?service=sync&context=fx_desktop_v3&entrypoint=mozilla.org-fx.wnp64&utm_content=%2Ffirefox%2FWNP%2F64%2F&utm_campaign=wnp64&utm_source=wnp-64" class="mzp-c-button mzp-t-download">{{ button_create }}</a><br>
+    <span class="wn64-signin">{{ login_desc }} <a href="https://accounts.firefox.com/signin">{{ login_link }}</a></span>
```

Comment:
There is a problem here, this is missing query params, we must specify query params for sync, so users can login to sync. cc @craigcook @stephaniehobson